backend/Dockerfile.driver

```diff
@@ -16,8 +16,8 @@
 ARG SOURCE_CODE=.
 
 # Use ubi8/nodejs-14 as base image
-#@follow_tag(registry.access.redhat.com/ubi8/go-toolset:1.20)
-FROM registry.access.redhat.com/ubi8/go-toolset:1.20 as builder
+#@follow_tag(registry.access.redhat.com/ubi8/go-toolset:1.21)
```

Comment:
@follow_tags have been removed from midstream, so we should do it here too

Reply:
Done, removed tags.